docs/topics/forms/fieldsets/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Group form fields
+layout: default
+parent: Web forms
+nav_order: 5
+description: Group form fields that belong together inside a fieldset with a descriptive legend.
+---
+
+# Fieldset and legend to group form fields
+
+Group form fields that belong together inside a `<fieldset>` with a descriptive `<legend>`. The advantage of this is that the fields are  visually and in the code connected to each other. The <legend> serves as the name of the group.
```

Comment:
"The advantage to this is that the fields are grouped together both visually and semantically, making the design match the code."